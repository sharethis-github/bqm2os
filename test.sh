#!/bin/bash

set -e

cd $(dirname $0)

. env.sh

docker build -t $imagename:$current_commit .

echo Executing unit tests
./unit-test.sh

echo Executing integration tests - $(date)
echo
./int-test.sh
