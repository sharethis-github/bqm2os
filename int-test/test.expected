atest2 depends on nothing
atest2.before depends on atest2 atest2.one
atest2.complex_json_data depends on atest2
atest2.extract_test_localdata_comma depends on atest2 atest2.test_local_tabbed_data
atest2.extract_test_localdata_comma_no_header depends on atest2 atest2.test_local_tabbed_data
atest2.extract_test_localdata_default depends on atest2 atest2.test_local_tabbed_data
atest2.extract_test_localdata_json depends on atest2 atest2.test_local_tabbed_data
atest2.extract_test_localdata_tab depends on atest2 atest2.test_local_tabbed_data
atest2.load_three depends on extract.atest2.three
atest2.mergelog_mini_sample depends on atest2
atest2.one depends on atest2 atest2.three atest2.two
atest2.read_load_thread depends on atest2 atest2.load_three
atest2.test_local_json_data depends on atest2
atest2.test_local_tabbed_data depends on atest2
atest2.three depends on atest2
atest2.two depends on atest2
atest2.url_expanded_kw depends on atest2 atest2.mergelog_mini_sample
extract.atest2.extract_test_localdata_comma depends on atest2.extract_test_localdata_comma
extract.atest2.extract_test_localdata_comma_no_header depends on atest2.extract_test_localdata_comma_no_header
extract.atest2.extract_test_localdata_default depends on atest2.extract_test_localdata_default
extract.atest2.extract_test_localdata_json depends on atest2.extract_test_localdata_json
extract.atest2.extract_test_localdata_tab depends on atest2.extract_test_localdata_tab
extract.atest2.three depends on atest2.three
